ThunderCloud/AppCollectionCell.swift

```diff
@@ -26,6 +26,15 @@ extension AppCollectionItem: CollectionCellDisplayable {
         }
         return UIApplication.shared.canOpenURL(launchURL)
     }
+    
+    public var accessibilityLabel: String? {
+        let params = [
+            "APP_NAME": itemTitle ?? "Unknown".localised(with: "_APP_NAME_UNKNOWN")
+        ]
+        return enabled ?
+            "{APP_NAME}. Installed.".localised(with: "_APP_COLLECTION_ITEM_INSTALLED", paramDictionary: params) :
```

Comment:
Are there meant to be full stops after `{APP_NAME}`? Is this so when it's read out, there's a pause perhaps? :)

Reply:
@ryanbourneuk there is aye :D